### ISSUE TITLE
 Confirmations controller route error not found

### ISSUE THREAD

**Issue by igorgesso** [lynndylanhurley/devise_token_auth#1316]
**rails (5.2.3)
devise_token_auth (1.1.0)
devise (4.6.2)**

I'm trying to make the confirmation part of the devise auth token work, but I'm running into a problem.

When accessing the link sent in the email always goes to the exception in the confirmations_controller as it is in the log and the route is not found.

I tried to understand the confirmations_controller to see what could be the problem, if anyone can give me some hint of what I might be doing wrong.

sorry for my english translate.

[Confirmation.log](https://github.com/lynndylanhurley/devise_token_auth/files/3407768/Confirmation.log)

**Comment by igorgesso** [comment]
can someone explain how is the confirmable flow type what has in faq for reset password flow? You don't need as many details as faq.

**Comment by experimatt** [comment]
@igorgesso did you ever figure out how to prevent this error? I'm having the exact same problem, and can't find good documentation on how the user email confirmation flow should work. Like which parameters are required (e.g. `email` and `redirect_url`), and which endpoint to hit (e.g. POST to `/auth/confirmation`).

It clearly works to _create_ the confirmation request (which sends an email), but upon clicking the link, I get the same NotFound error.

```
9:18:40 PM api.1 |  Started GET "/api/auth/confirmation?config=default&confirmation_token=eyY53XBy2sn1vg5DKC5u&redirect_url=http%3A%2F%2Flocalhost%3A3000%2Faccount" for ::1 at 2020-01-29 21:18:40 -0600
9:18:40 PM api.1 |  Processing by DeviseTokenAuth::ConfirmationsController#show as HTML
9:18:40 PM api.1 |    Parameters: {"config"=>"default", "confirmation_token"=>"eyY53XBy2sn1vg5DKC5u", "redirect_url"=>"http://localhost:3000/account"}
9:18:40 PM api.1 |  Unpermitted parameters: :config, :redirect_url
9:18:40 PM api.1 |  Completed 404 Not Found in 8ms (ActiveRecord: 1.2ms)
9:18:40 PM api.1 |  ActionController::RoutingError (Not Found):
9:18:40 PM api.1 |  devise_token_auth (1.1.3) app/controllers/devise_token_auth/confirmations_controller.rb:28:in `show'
```

**Comment by experimatt** [comment]
I just figured out what caused this - I was trying to confirm an email that was already confirmed.

Hopefully https://github.com/lynndylanhurley/devise_token_auth/issues/1123 will resolve this issue.

**Comment by kaka-ruto** [comment]
Will be good if this is mentioned in the docs as well @experimatt

**Comment by MaicolBen** [comment]
was it resolved by #1557 ?
